ros_ethercat_model/include/ros_ethercat_model/hardware_interface.hpp

```diff
@@ -37,6 +37,8 @@
 
 #include <string>
 #include <vector>
+#include <hardware_interface/actuator_state_interface.h>
```

Comment:
Why is this needed here?
It probably should be inside of `actuator_command_interface.h` instead.

Reply:
hmm, good point, it isn't needed here... removing